sdk/tables/azure-data-tables/src/test/java/com/azure/data/tables/TableAsyncClientTest.java

```diff
@@ -1009,4 +1012,30 @@ public void canUseSasTokenToCreateValidTableClient() {
             .expectComplete()
             .verify();
     }
+
+    @Test
+    public void setAndListAccessPolicies() {
```

Comment:
Let's add a test with multiple `SignedIdentifier`s to ensure there isn't any issues when multiple are returned.